cdap-app-templates/cdap-etl/cdap-data-pipeline/src/test/java/co/cask/cdap/datapipeline/DataPipelineTest.java

```diff
@@ -396,6 +399,457 @@ private void testErrorTransform(Engine engine) throws Exception {
   }
 
   @Test
+  public void testIntToStringConversion() throws Exception {
+    Gson GSON = new GsonBuilder()
+      .registerTypeAdapterFactory(new CaseInsensitiveEnumTypeAdapterFactory())
+      .create();
+
+    String configJsonNoQuote =
```

Comment:
These strings are generated using the 'toJson' method. We have to explicitly define the strings because we are checking for quoted and unquoted numbers .i.e. we have to check both 7 and "7" and there's no clean way to do it without putting in the whole json itself.

You're right about the length. I'll see if I can generate a shorter example, but the strings will still be there.